### DISCUSSION TITLE
[Feature request] Please distribute the program as a universal pkg

### DISCUSSION THREAD

**Discussion by Twig6943** [ivangabriele/clamav-desktop#173] [Feature Request Proposals]
Distributing as an .AppImage or a flatpak would be better cuz everybody could use the same pkg that way. (Less issues to worry about in a way)

**Comment by ivangabriele** [comment]
For the AppImage it's already answered there: #161. As for Flatpak, I need to check if they allow elevated permissions and cross-dependencies.

I keep it here but it's a premature discussion for now. I first need to finish the first real release. I won't have time to handle that before it's done.

Unless any kind contributor wants to pick this subject ^^.

**Reply by bojogon7** [reply]
+1 for flatpak
ClamTK and ClamUI are available as flatpak on flathub
